features/openhab-addons-external/src/main/resources/conf/panasonictv.cfg

```diff
@@ -0,0 +1,4 @@
+################################ Panasonic TV Binding #######################################
+#
+# IP address of a Panasonic TV instance
+# <id>=
```

Comment:
A commented out example would probably be very helpful here.  Also, the `##### Panasonic TV Binding #####` heading isn't needed.